core/scripts/chaincli/handler/keeper_verifiable_load.go

```diff
@@ -0,0 +1,232 @@
+package handler
+
+import (
+	"context"
+	"log"
+	"math/big"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/montanaflynn/stats"
+
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/verifiable_load_upkeep_wrapper"
+)
+
+const (
+	// WorkerNum is the total number of workers calculating upkeeps' delay summary
+	WorkerNum = 5
+	// RetryDelay is the time the go routine will wait before calling the same contract function
+	RetryDelay = 1 * time.Second
+	// RetryNum defines how many times the go routine will attempt the same contract call
+	RetryNum = 3
+	// MaxUpkeepNum defines the size of channels. Increase if there are lots of upkeeps.
+	MaxUpkeepNum = 100
```

Comment:
I would have all of these consts as unexported if possible?

Reply:
done